sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -670,27 +670,33 @@ def download_blob(self, offset=None, length=None, validate_content=False, **kwar
         }
         return StorageStreamDownloader(extra_properties=extra_properties, **options)
 
-    def _delete_blob_options(self, delete_snapshots=False, **kwargs):
+    @staticmethod
+    def _generic_delete_blob_options(delete_snapshots=False, **kwargs):
```

Comment:
Maybe we want to do delete_snapshots=None?

Reply:
That's a private method, and this PR is not about changing the behavior of the main "delete" behavior but to add batch support. I won't suggest to do that here, assuming we want to do it.

My two cents is this won't change a thing, since we interpret this value in a boolean way, so `None` and `False` would act the same.